docs/release-notes/deploy-local-auto-password.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+**Security Fixes**
+
+-  CLI: If no user accounts have already been created, and neither
+   ``security.initial_user_password`` in master.yaml nor ``--initial-user-password`` is present when
+   running ``det deploy local`` with the ``master-up`` or ``cluster-up`` commands, an initial
+   password will be generated and displayed to the user.
```

Comment:
not sure if this is what is intended:

:orphan:

**Security Fixes**

-  CLI: If no user accounts have already been created, and neither
   ``security.initial_user_password`` in master.yaml nor the ``--initial-user-password`` is specified when
   running ``det deploy local`` with the ``master-up`` or ``cluster-up`` commands, an initial
   password will be generated and displayed to the user.

Reply:
Would you mind clarifying what the proposed change is? Just the addition of the word "the" or was there something else I missed? I would probably say "the `--initial-user-password` CLI flag" in that case, to be extra clear.